pepper-apis/docs/specification/src/endpoints/user.studies.answers.yml

```diff
@@ -0,0 +1,37 @@
+parameters:
+  - in: path
+    name: user
+    required: true
+    description: the participant's guid
+    schema:
+      type: string
+  - in: path
+    name: study
+    required: true
+    description: the study's unique identifier
+    schema:
+      type: string
+  - in: path
+    name: stableId
+    required: true
+    description: the question's stableId
+    schema:
+      type: string
+
+get:
+  operationId: getAnswer
+  tags:
+    - Activities
+  summary: get an answer for a specific question by stableId
+  description: |
+    Gets an answer submissions
+
+    _This API does NOT allow access to temporary users._
+  responses:
+    200:
+      content:
+        application/json:
+          schema:
+            $ref: '../pepper.yml#/components/schemas/Answer.Composite'
```

Comment:
Does this endpoint support all answer types or just the `COMPOSITE` answer type?

Reply:
All answer types

Reply:
Updated the list of schemas